accounts-db/src/accounts_db/load_accounts_histogram.rs

```diff
@@ -0,0 +1,123 @@
+use {
+    hdrhistogram::Histogram,
+    std::time::{Duration, Instant},
+};
+
+#[derive(Debug)]
+pub struct LoadAccountsHistogram {
+    samples: Vec<Duration>,
+    previous_submit: Instant,
+}
+
+impl LoadAccountsHistogram {
+    const SUBMIT_INTERVAL: Duration = Duration::from_secs(60);
+
+    #[must_use]
+    pub fn new() -> Self {
+        Self {
+            samples: Vec::new(),
+            previous_submit: Instant::now(),
+        }
+    }
+
+    /// Records `sample`
+    ///
+    /// Will be included in the next submission
+    pub fn record(&mut self, sample: Duration) {
+        self.samples.push(sample);
+    }
+
+    /// Submits datapoint if enough time has passed since previous submission
+    pub fn maybe_submit(&mut self) {
+        if self.previous_submit.elapsed() >= Self::SUBMIT_INTERVAL {
+            self.submit();
+        }
+    }
+
+    /// Submits datapoint
+    pub fn submit(&mut self) {
+        let duration = self.previous_submit.elapsed(); // do this before building the histogram
+        let samples = std::mem::take(&mut self.samples); // use `take` because we want to clear `samples`
+        let histogram = Self::build_histogram(&samples);
+
+        datapoint_info!(
```

Comment:
I'd assume it wouldn't happen, but if there are no samples do we still want to report this?